contrib/gdk-pixbuf/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ if(AVIF_BUILD_GDK_PIXBUF)
             add_library(pixbufloader-avif MODULE ${GDK_PIXBUF_SRCS})
 
             target_link_libraries(pixbufloader-avif PUBLIC ${GDK_PIXBUF_LIBRARIES} avif)
+            target_link_directories(pixbufloader-avif PUBLIC ${GDK_PIXBUF_LIBRARY_DIRS})
```

Comment:
@novomesk Daniel, could you take a look at this change?

Reply:
OK for me.